discoveryApiResearchNowApiMapping.md

```diff
@@ -0,0 +1,445 @@
+* `filters` string
+  * "owner", "subjectLiteral", "holdingLocation", "deliveryLocation", "language", "materialType", "mediaType", "carrierType", "publisher", "contributor", "creator", "issuance", "createdYear", "dateAfter"', or "dateBefore"
+  * Specify a hash of filters to apply, where keys are from terms above
+
+## ResearchNow API
+[Github](https://github.com/NYPL/sfr-ingest-pipeline/tree/development/app/sfr-search-api#searching)
+[GET `/v0.1/research-now/v3/search-api`](https://dev-platformdocs.nypl.org/#/research-now/get_v0_1_research_now_v3_search_api)
+[POST `/v0.1/research-now/v3/search-api`](https://dev-platformdocs.nypl.org/#/research-now/post_v0_1_research_now_v3_search_api)
+### Parameters
+* `queries` array of objects
+  * Objects for the queries are formatted as `{"field": filter, "value": value}`
+* `filters` array of objects
+  * Objects for the filters are formatted as `{"filter": filter, "value": value}`
+* `field` string
+  * "keyword", "title", "author", "standardNumber" (ISBN, ISSN, LCCN and OCLC) and "subject"
+* `query` string
+* `recordType`
+  * Internal record type to return with the work. Either instances or editions.
+* `page` integer
+* `per_page` integer
+* `sort` array of objects
+  * Objects are formatted as `{"field": field, "dir": dir}`
+* `language`
+* `years`
+  * This should be formatted as `{"start": year, "end": year}`.
+
+For the DRBB/SCC integration, the DRBB data is fetched using a POST request with a body containing `page`, `per_page`, `filters` (years, languages), and `queries` (author/contributor, subject).
+
+<!-- <style>
+  table ul {
+    list-style: none;
+  }
+</style> -->
+
+## API Parameters Comparison
+This table lists, in the first column, the frontend features related to searching and search filters available from the frontend. The 2nd and 3rd column list out the Discovery API and ResearchNow API parameters that correlate to that frontend functionality.
+
+Single quotes (') are used for frontend terminology. Italics are used to describe frontend features. Code styling (e.g. `filters`) is used for the parameters the respective APIs except.
+
+|Discovery front end|Discovery API |ResearchNow API|
+|-------------------|--------------|---------------|
+|_Search field_ |`q`           |`queries`, "value"       |
+|_Search field dropdown_|`search_scope`|`queries`, "field"|
+|<ul>'All Fields'   |<ul>"all"     |<ul>"keyword"|
+|<ul>'Title'       |<ul>"title"  |<ul>"title"       |
+|<ul>'Author/Contributor'|<ul>"contributor" |<ul>"author"|
+|<ul>'Standard Number'|<ul>"standard_number"|<ul> "standardNumber"|
+|_Search page filters_|`filters`     ||
+|<ul>'Format'       |<ul>`filters[materialType]`|<ul>N/A*|
+|<ul>'Date':<ul><li>'Start Year'<li>'End Year'| <ul><br>`filters[dateAfter]`<br>`filters[dateBefore]` |`filters`<br><ul>`years[start]`<br>`years[end]`|
+|<ul>'Language'|<ul>`filters[language]`|<ul>`language`|
+|_Filters linked to from a bib page_|
```

Comment:
To query author/subject in this way we don't treat them as a filter, but rather as a query. I'm not sure how you've implemented it here, but I feel like the crosswalk is a little unclear here regarding that.